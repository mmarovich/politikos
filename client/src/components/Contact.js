import React, { Component } from 'react'

import '../../styles/Contact.css'

export default class Contact extends Component {
    render() {
        return(
            <div className="contact-container">
                <h3>Questions? Suggestions? Concerns?</h3>
                <p>email: <a href="mailto:politikosservice@gmail.com">politikosservice@gmail.com</a></p>
            </div>
        )
    }
}