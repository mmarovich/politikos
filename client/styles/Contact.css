.contact-container {
    padding: 30px;
    text-align: center;
    min-height: calc(100% - 94px - 56px);
    margin: 30px auto 0;
}

.contact-container > h3 {
    padding: 30px;
}