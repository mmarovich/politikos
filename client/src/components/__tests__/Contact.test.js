import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router'
import Contact from '../Contact'

describe('Contact component', () => {
    it('render Contact', () => {

        const tree = renderer.create(
            <MemoryRouter>
                <Contact />
            </MemoryRouter>
        ).toJSON()

        expect(tree).toMatchSnapshot()
    })
})