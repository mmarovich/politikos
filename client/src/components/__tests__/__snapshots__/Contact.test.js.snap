exports[`Contact component render Contact 1`] = `
<div
  className="contact-container">
  <h3>
    Questions? Suggestions? Concerns?
  </h3>
  <p>
    email: 
    <a
      href="mailto:politikosservice@gmail.com">
      politikosservice@gmail.com
    </a>
  </p>
</div>
`;
